compiler/rustc_expand/src/mbe/transcribe.rs

```diff
@@ -752,39 +763,37 @@ fn transcribe_metavar_expr<'a>(
 }
 
 /// Extracts an metavariable symbol that can be an identifier, a token tree or a literal.
-fn extract_var_symbol<'a>(
+fn extract_symbol_from_pnr<'a>(
```

Comment:
The jargon is getting a bit obscure. It took me a while to figure what a `pnr` is. "Parc Naturel Régional"?

Reply:
Non, ce n'est pas "Parc Naturel Régional" :)

`pnr` is in the function signature and `extract_symbol_from_parse_nt_result` would be too long IMO.